sim/paladin/righteous_fury.go

```diff
@@ -0,0 +1,71 @@
+package paladin
+
+import (
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+)
+
+func (paladin *Paladin) registerRighteousFury() {
+	if !paladin.Options.RighteousFury {
+		return
+	}
+	horRune := proto.PaladinRune_RuneHandsHandOfReckoning
+	hasHoR := paladin.hasRune(horRune)
+	actionID := core.ActionID{SpellID: core.TernaryInt32(hasHoR, int32(horRune), 25780)}
+
+	rfThreatMultiplier := 0.6 + core.TernaryFloat64(hasHoR, 0.2, 0.0)
+	// Improved Righteous Fury is multiplicative.
+	rfThreatMultiplier *= 1.0 + []float64{0.0, 0.16, 0.33, 0.5}[paladin.Talents.ImprovedRighteousFury]
+
+	paladin.OnSpellRegistered(func(spell *core.Spell) {
+		if spell.SpellSchool.Matches(core.SpellSchoolHoly) {
+			spell.ThreatMultiplier *= 1.0 + rfThreatMultiplier
+		}
+	})
+
+	if !hasHoR { // This is just a visual/UI indicator when we don't have HoR rune.
+		paladin.RegisterAura(core.Aura{
+			Label:    "Righetous Fury",
+			ActionID: actionID,
+			Duration: core.NeverExpires, // 30 minutes without HoR rune, but no need to model
+			OnReset: func(aura *core.Aura, sim *core.Simulation) {
+				aura.Activate(sim)
+			},
```

Comment:
Knit: Can you wrap this aura with `core.MakePermanent` instead? It's a nice utility for passive auras like this